doc/ap_verify/index.rst

```diff
@@ -0,0 +1,27 @@
+.. _ap_verify-package:
+
+#########
+ap_verify
+#########
+
+The ``ap_verify`` package wraps :py:mod:`ap_pipe <lsst.ap.pipe>` with support for managing :py:mod:`verify <lsst.verify>` metrics.
```

Comment:
For "verify" use the actual module namespace for clarity:

```rst
for managing `lsst.verify` metrics.
```

Note that single back-ticks can be used for any type of Python API reference (being the default role).

Reply:
So `:mod:` is also redundant?